src/components/ui/EditForm/index.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import Button from '../Button';
+import EditIcon from '../EditIcon';
+import { convertFromPixelsToRem } from '../../../misc';
+import { inputValidation } from 'lib/validator';
```

Comment:
You are still doing this here. Can you add path aliasing to all the rest?

Reply:
resolved

Reply:
I didn't initially understand how this part aliasing thing work.

Reply:
Glad you now do.